src/components/password/password.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useRef } from "react";
+import { TextboxProps } from "../textbox";
+import { StyledPassword, HiddenAriaLive } from "./password.style";
+import guid from "../../__internal__/utils/helpers/guid";
+import useLocale from "../../hooks/__internal__/useLocale";
+
+export type TypeOptions = "text" | "password";
+
+export interface PasswordProps extends TextboxProps {
+  /** HTML type attribute of the input */
+  type?: TypeOptions;
+}
+
+export const Password = ({ id, type, ...rest }: PasswordProps) => {
+  const internalInputId = useRef(id || guid());
+  const l = useLocale();
+  return (
+    <>
+      <StyledPassword
+        data-element="styled-password-container"
+        id={internalInputId.current}
+        autoComplete="off"
+        type={type || "password"}
+        {...rest}
+      />
+      {/* Below hidden aria live <p> may be superseded by 
+       ongoing work to create a dedicated aria live
+       component - FE-5685  */}
+      <HiddenAriaLive role="status" aria-live="polite">
+        {type === "text"
+          ? l.password.ariaLiveShownMessage()
+          : l.password.ariaLiveHiddenMessage()}
+      </HiddenAriaLive>
```

Comment:
Nice work on the live region stuff @tomdavies73 👍🏼 

This looks good to me, though I'm not sure if we need to add a small delay on rendering the message, as I believe there are some cases where the content update is too quick for a screen reader so it will duplicate the voice announcement. 

However, I can't say this with certainty, so I will approve for now and perhaps this is something @DlgSHi will be able to verify for you? 😄